frontend/packages/kubevirt-plugin/integration-tests/tests/vm.overview.scenario.ts

```diff
@@ -0,0 +1,123 @@
+import * as _ from 'lodash';
+import { testName } from '../../../../integration-tests/protractor.conf';
+import { isLoaded, resourceTitle } from '../../../../integration-tests/views/crud.view';
+import * as vmView from '../views/virtualMachine.view';
+import {
+  asyncForEach,
+  createResource,
+  deleteResource,
+} from '../../../console-shared/src/test-utils/utils';
+import { getVmManifest, basicVmConfig } from './utils/mocks';
+import { exposeServices } from './utils/utils';
+import { VirtualMachine } from './models/virtualMachine';
+import { TABS, DASH, VM_BOOTUP_TIMEOUT_SECS } from './utils/consts';
+import { NodePortService } from './utils/types';
+
+describe('Test vm overview', () => {
+  const vmName = `vm-${testName}`;
+  const cloudInit = `#cloud-config\nuser: cloud-user\npassword: atomic\nchpasswd: {expire: False}`;
+  const serviceCommon = { name: vmName, kind: 'vm', type: 'NodePort', namespace: testName };
+  const testVm = getVmManifest('Container', testName, vmName, cloudInit);
+  const vm = new VirtualMachine(testVm.metadata);
+  const nodePortServices = new Set<NodePortService>();
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-ssh`,
+    port: '22',
+    targetPort: '20022',
+  });
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-smtp`,
+    port: '25',
+    targetPort: '20025',
+  });
+  nodePortServices.add({
+    ...serviceCommon,
+    exposeName: `${vmName}-service-http`,
+    port: '80',
+    targetPort: '20080',
+  });
+
+  beforeAll(() => {
+    createResource(testVm);
+    exposeServices(nodePortServices);
+  });
+
+  afterAll(() => {
+    deleteResource(testVm);
+  });
+
+  beforeEach(async () => {
+    await vm.navigateToTab(TABS.OVERVIEW);
+    await isLoaded();
+  });
+
+  it('Check vm details in overview when vm is off', async () => {
+    const expectation = {
+      name: vmName,
+      description: testName,
+      os: basicVmConfig.operatingSystem,
+      profile: basicVmConfig.workloadProfile,
+      template: 'Template\nTrhel7-desktop-tiny',
```

Comment:
IMO, it would look nicer to select `<a />` element under the template description